.github/CODEOWNERS

```diff
@@ -3,3 +3,4 @@
 weave-js/src/common @wandb/fe-infra-reviewers
 weave-js/src/components @wandb/fe-infra-reviewers @wandb/weave-team
 weave-js/src/assets @wandb/fe-infra-reviewers @wandb/weave-team
+/weave_query/ @wandb/query-engine-reviewers @wandb/weave-team
```

Comment:
should there be a leading slash here?

Reply:
Yes, it's unlikely to make a difference in practice, but the leading slash captures my intention of only matching the top-level `weave-query` directory and not a `weave-query` directory anywhere in the repo.